src/InfoViz/Native/HistogramSelector/score.js

```diff
@@ -859,20 +865,22 @@ export default function init(inPublicAPI, inModel) {
   function rescaleDividers(paramName, oldRangeMin, oldRangeMax) {
     if (model.fieldData[paramName] && model.fieldData[paramName].hobj) {
       const def = model.fieldData[paramName];
-      const hobj = model.fieldData[paramName].hobj;
-      if (hobj.min !== oldRangeMin || hobj.max !== oldRangeMax) {
-        def.dividers.forEach((divider, index) => {
-          if (oldRangeMax === oldRangeMin) {
-            // space dividers evenly in the middle - i.e. punt.
-            divider.value = (((index + 1) / (def.dividers.length + 1)) *
-                             (hobj.max - hobj.min)) + hobj.min;
-          } else {
-            // this set the divider to hobj.min if the new hobj.min === hobj.max.
-            divider.value = (((divider.value - oldRangeMin) / (oldRangeMax - oldRangeMin)) *
-                             (hobj.max - hobj.min)) + hobj.min;
-          }
-        });
-        sendScores(def);
+      if (showScore(def)) {
```

Comment:
I think we put in this guard while working together, but now I don't remember why we needed it - add a comment?